src/components/settings/TranslatorMode.svelte

```diff
@@ -29,6 +30,11 @@
   }
 
   onMount(() => setTimeout(cleanUpMacros));
+
+  const autoPrefixRules = [{
+    error: 'You have no selected translation languages',
+    assert: (value) => !(/\$filterLang/gi.test(value)),
+  }];
```

Comment:
Rules are not needed in these settings anymore since `$filterLang`  always has a replacement.